### PR TITLE
2023.3.x feature auth bg image

### PR DESCRIPTION
**Pull request by KinyaElGrande**
Add support for personalized auth background screen .

### CODE REVIEW

File: auth/assets/templates/inc_header.html.tpl [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
+	<!-- Custom CSS -->
+	<link href="{{ links.AuthAssets }}/style.css?{{ buildtime }}" rel="stylesheet">
+	<title>Corteza</title>
+</head>
+{{ if .settings.BackgroundUI.BackgroundImageSrcUrl}}
+{{ $background := (printf `background: url("%s") no-repeat top; background-size: cover; background-attachment: fixed; %s` .settings.BackgroundUI.BackgroundImageSrcUrl .settings.BackgroundUI.Styles) }}
```
Comment (darh) [inline auth/assets/templates/inc_header.html.tpl]: this looks a bit much
why did you add styles here? why not inside <head><style>...</style></head>?

**Review by darh — CHANGES_REQUESTED** [review]

**Comment by KinyaElGrande** [comment]
[Auth background image](https://trello.com/c/FyrM0Z7o/3316-auth-background-image)
